.github/workflows/nightly.yml

```diff
@@ -17,9 +17,7 @@
 ################################################################################
 
 name: "flink-connector-aws: nightly build"
-on:
-  schedule:
-    - cron: "0 0 * * *"
+on: [push, pull_request]
```

Comment:
Is switching nightly by cron to on push, pr intended?

Reply:
It is only used to show the nightly CI can paIt after this pr. You can see the message of the commit related to this change.